src/votekit/pref_profile.py

```diff
@@ -63,27 +68,46 @@ def num_ballots(self) -> Fraction:
 
         return num_ballots
 
-    def to_dict(self, standardize: bool) -> dict:
+    def to_dict(self, standardize: bool = False) -> dict:
         """
-        Converts ballots to dictionary with rankings (keys) and the
+        If ranked ballots, converts ballots to dictionary with rankings (keys) and the
         corresponding total weights (values)
 
+        If PointBallots, converts to dictionary with keys candidates and values total points.
+
+        `standardize`: boolean, if True, normalizes weights by number of ballots
         Returns:
-            A dictionary with with ranking (keys) and corresponding total \n
+            A dictionary with with ranking/candidates (keys) and corresponding total \n
             weights (values)
         """
         num_ballots = self.num_ballots()
         di: dict = {}
-        for ballot in self.ballots:
-            rank_tuple = tuple(next(iter(item)) for item in ballot.ranking)
-            if standardize:
-                weight = ballot.weight / num_ballots
-            else:
+
+        if isinstance(self.ballots[0], Ballot):
```

Comment:
a lot of shared code between these two, moot if they merge, but I'd refactor this to avoid duplication if not